src/components/Paginator.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect, } from "react";
+import api from "../api.js";
+
+
+/**
+* Renders a list of items fetched from a given URL, paginating as appropriate
+* (pagination not built yet!)
+* @prop requestConf: Object - the axios config needed to request the items'
+* data
+* @prop ItemComponent: React Component - the component to be used when
+* rendering the data of each item.
+* @prop idKey: String - a key that every item has which uniquely identifies
+* that item
+* @prop itemsKey: String - the key on the response data where the list of
+* items is stored
+*/
```

Comment:
Useful note to self but does clutter up the file a bit which we should avoid on PRs

Reply:
That's not a note to self, that's documentation